wp-job-manager-template.php

```diff
@@ -1332,11 +1372,12 @@ function the_job_salary( $before = '', $after = '', $echo = true, $post = null )
 	 * @param WP_Post $post
 	 * @param string  $before
 	 * @param string  $salary
+	 * @param string  $salarymax
 	 * @param string  $currency
 	 * @param string  $unit
 	 * @param string  $after
 	 */
-	$job_salary = apply_filters( 'the_job_salary_message', $job_salary, $post, $before, $salary, $currency, $unit, $after );
+	$job_salary = apply_filters( 'the_job_salary_message', $job_salary, $post, $before, $salary, $currency, $salarymax, $unit, $after );
```

Comment:
This might break existing integrations

Reply:
Good question, I don't have an empirical solution to that, it can be anything from adding the param after $after, to changing $salary to be a concatenation of $min . '-' . $max, if it's a salary range .
I'd prefer the second option, as $salary is documented as string, it wouldn't break the contract, @gkaragia do you have an idea ?

Reply:
I think appending it as an extra argument in the end is the safest. 

If we don't add it as an extra argument and concatenate instead, this will break existing sites that use the `$after` function argument or specify a `$unit`. In these cases the job salary mind end up with weird values in the filter.